docs/request-for-comment/README.md

```diff
@@ -0,0 +1,30 @@
+# What is an RFC?
+An RFC is a [Request for Comments](https://en.wikipedia.org/wiki/Request_for_Comments)
+
+An RFC is a reflection of the team's thoughts and plans _at that moment_.
+
+# Why do we use RFCs?
+We write an RFC to *communicate intent and get feedback* before we start building something. [RFC driven development](https://engineering-management.space/post/rfc-driven-development/) is a good concept to consider as we build for the Platform.
+
+# Who writes an RFC?
+You! Anyone on a team, including product managers, UX, research, as well as engineers.
+
+# When do I write an RFC?
+Anytime! Whenever your team is making a decision that could affect the platform, write the details and get feedback from others on the Platform.
+
+# How do I create an RFC and manage its lifecycle?
```

Comment:
In the template, there is a status that is set to PROPOSED, what are the other valid status choices and how does one determine setting from one to the other?